server/views/pages/overview/partials/supportNeedsTab/supportNeedsTabContents.test.ts

```diff
@@ -1,54 +1,101 @@
-import * as fs from 'fs'
 import * as cheerio from 'cheerio'
-import { Cheerio, CheerioAPI } from 'cheerio'
-import nunjucks, { Template } from 'nunjucks'
-import { registerNunjucks } from '../../../../../utils/nunjucksSetup'
+import nunjucks from 'nunjucks'
 import aValidPrisonerSupportNeeds from '../../../../../testsupport/supportNeedsTestDataBuilder'
 import aValidPrisonerSummary from '../../../../../testsupport/prisonerSummaryTestDataBuilder'
+import fallbackMessageFilter from '../../../../../filters/fallbackMessageFilter'
+import formatDateFilter from '../../../../../filters/formatDateFilter'
 
-describe('Support Needs tab view', () => {
-  const template = fs.readFileSync('server/views/pages/overview/partials/supportNeedsTab/supportNeedsTabContents.njk')
-  const prisonerSummary = aValidPrisonerSummary()
-
-  let compiledTemplate: Template
-  let viewContext: Record<string, unknown>
+const njkEnv = nunjucks.configure([
```

Comment:
👍 Excellent refactoring of this test so that it sets up the nunjucks env in the way our more recent tests do. Nice